lab-brianna/lib/storage.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const debug = require('debug')('http:storage');
+const storage = {};
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'});
```

Comment:
**TODO FOR LOST POINTS**

Your tests broke at line 5 because you didn't require in some extra dependencies.